changelog.d/6278.misc

```diff
@@ -0,0 +1 @@
+Update log line to not use an undefined variable.
```

Comment:
Bugfix? Broadly please remember that changelogs are for consumption by server admins, so should describe what's being fixed.

```suggestion
Fix exception when remote servers attempt to join a room that they're not allowed to join.
```